flipperdevices/Flipper-Android-App
# FileManager selection, renaming, file options

**Background**

Add more features to file manager

**Changes**

- Selection
- Multiple deletions
- Rename(aka move)
- Fix uploading doesn't start(replaced SharedFlow with Channel)
- Improve OrangeAppBar
- BottomSheet options for file/folder

**Test plan**

- Open bridge connection sample
- Try click `more` button
- Try click select
- Use new options and see cool animations
